Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp

```diff
@@ -4526,7 +4520,7 @@ auto OMGIRGenerator::emitCatchTableImpl(ControlData& data, const ControlData::Tr
     }
 
     if (target.type == CatchKind::CatchRef || target.type == CatchKind::CatchAllRef) {
-        Variable* var = m_proc.addVariable(pointerType());
+        Variable* var = m_proc.addVariable(Int64);
```

Comment:
Maybe we want a wasmRefType() helper for this?

Reply:
Added such a helper to `B3Type.h` and changed it here. Should I additionally use this abstraction in other places where a wasm ref type is meant?